### ISSUE TITLE
any model with quadruplet loss

### ISSUE THREAD

**Issue by jpainam** [michuanhaohao/reid-strong-baseline#38]
Hi, thank you for the strong baseline. Do you have any baseline using quadruplet loss. The implementation as described in the paper is a bit tricky and the authors didn't release their implementation.
Thank you

**Comment by michuanhaohao** [comment]
@jpainam No. I did many experiments with quadruplet loss. I do not think quadruplet loss is better than triplet loss.

**Comment by jpainam** [comment]
@michuanhaohao pls, can you paste here the nn.Module of the quadruplet loss you did experiments with? I'm also facing some difficulty to make it work.
Thank you

**Comment by michuanhaohao** [comment]
@jpainam Quad loss is not compatible for this repo. It need a new data provider. And i did experiments 1~2 years ago. I do not have the codes now.

**Comment by jpainam** [comment]
Thank you. By the way, congrats for your Pattern Recognition publication.
